apps/backoffice-v2/src/common/utils/get-severity-from-risk-score.ts

```diff
@@ -0,0 +1,17 @@
+import { Severity, TSeverity } from '@/common/types';
+
+export const getSeverityFromRiskScore = (riskScore: number): TSeverity => {
+  if (riskScore <= 39) {
+    return Severity.LOW;
+  }
+
+  if (riskScore <= 69) {
+    return Severity.MEDIUM;
+  }
+
+  if (riskScore <= 84) {
+    return Severity.HIGH;
+  }
+
+  return Severity.HIGH;
+};
```

Comment:
Also from [PR 1644](https://github.com/ballerine-io/ballerine/pull/2343), just moved to common.